testsuite/scripts/tutorials/test_charged_system.py

```diff
@@ -19,7 +19,7 @@
 import importlib_wrapper
 
 tutorial, skipIfMissingFeatures = importlib_wrapper.configure_and_import(
-    "@TUTORIALS_DIR@/charged_system/charged_system-1.py", N_SAMPLES=2,
+    "@TUTORIALS_DIR@/charged_system/charged_system.py", N_SAMPLES=2,
```

Comment:
don't we plan on writing a part 2 soon? #3955

Reply:
it doesn't matter, for now we do not have a part 2